node/directives/helper.ts

```diff
@@ -138,3 +138,34 @@ export const validateStoreToken = async (
 
   return { hasStoreToken, hasValidStoreToken, hasCurrentValidStoreToken }
 }
+
+export const validateAdminTokenOnHeader = async (
+  context: Context
+): Promise<{
+  hasAdminToken: boolean
+  hasValidAdminToken: boolean
+  hasCurrentValidAdminToken: boolean
```

Comment:
I would add `...OnHeader` to the name of these for clarity.

Reply:
Done